src/components/atoms/Page/Component.tsx

```diff
@@ -10,68 +10,72 @@ import {
   ScrollView,
   TouchableOpacity,
   KeyboardAvoidingView,
+  KeyboardAvoidingViewProps,
+  ViewProps,
 } from 'react-native';
 
-export type PageProps = View &
-  KeyboardAvoidingView & {
-    scrollEnabled?: boolean;
-    children?: React.ReactNode;
-    SafeAreaView: any;
-    style?: StyleProp<ViewStyle>;
-    backgroundColor?: string;
-    statusBar?: 'light-content' | 'dark-content';
-    unsafe?: boolean;
-    hidden?: boolean;
-    statusColor?: string;
-    safeArea?: boolean;
-    /**
-     * Enable to draw behind status bar android
-     * @default false
-     */
-    draw?: boolean;
+import type { SafeAreaView } from 'react-native-safe-area-context'
```

Comment:
@jefrydco don't used it directly here

Reply:
and SafeAreaView: typeof SafeAreaView;
canbe View or any component note typeof SafeAreaView